mesh/src/vertex/HoneycombVertexMeshGenerator.cpp

```diff
@@ -34,6 +34,7 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 */
 
 #include "HoneycombVertexMeshGenerator.hpp"
+#include "CheckpointArchiveTypes.hpp"
```

Comment:
I think this include should only feature in things actually making archives, so generally tests, and the odd "archiver" class?

Reply:
Resolved by commit 69c7ae2.